### ISSUE TITLE
Tournaments bugged

### ISSUE THREAD

**Issue by Bestle** [alkarinv/BattleArena#256]
Couldn't open the previous issue, so posting a new one.

Tournament bugged again, though it made it past the first round this time which is an improvement!

http://pastebin.com/8BBTKXrp

**Comment by alkarinv** [comment]
Damn, and I thought I had it all worked out.  Same issue but just hung after the first round?

**Comment by Bestle** [comment]
Yep it got past the first round but then hung completely.

Here are some more logs from that tourney if they are useful at all:

http://pastebin.com/eCCRTS4P

**Comment by Bestle** [comment]
Tried the latest version, still has some strange quirks.

[21:52:26 INFO]: [BattleArena] Tournament found a match with no players, cancelling tournament
[21:52:26 INFO]: Event was cancelled

It broadcast to all the players that the event was cancelled, but it still carried on?

**Comment by Bestle** [comment]
It seems that only the matches already in progress carried on, but the event did actually get cancelled.

This is incredibly frustrating ;p

**Comment by alkarinv** [comment]
Definitely.. What's your ip? I want to get this all figured out, but I'm having trouble recreating the problem.

**Comment by Bestle** [comment]
pvp.gontroller.com

**Comment by alkarinv** [comment]
Hey. I logged in but didn't see you.  Also your teamspeak has a password? Should it? or can you email me the password at alkarin.v@gmail.com.
Just want to be able to talk with ya.

**Comment by Bestle** [comment]
Tried out the latest version and things are super weird...

http://www.gontroller.com/caps/nHx4bbBTc.png

The team size is set to 1, but there are teams of 2 players taking part.

**Comment by Bestle** [comment]
Getting even weirder, halfway through the tournament this message came up in chat:

http://www.gontroller.com/caps/O8Q028x04.png

The tournament was already in progress so nobody could join, wtf.

**Comment by alkarinv** [comment]
Hey Bestle, I'm on right now on skype and Teamspeak.

**Comment by Bestle** [comment]
Not sure if you had finished working on them yet, but tournaments are still broken in the latest version on github.

This time the event just randomly cancelled itself.

**Comment by alkarinv** [comment]
Gah!.  And here I thought major progress had been made.

Unfortunately I'm on vacation now so it will remain broken for a couple more days -_-....

**Comment by Bestle** [comment]
Any progress on this? :)
